packages/admin-e2e-tests/src/pages/BasePage.ts

```diff
@@ -132,6 +132,10 @@ export abstract class BasePage {
 		await this.goto( this.url );
 	}
 
+	async waitForTimeout( timeout: number ) {
+		await new Promise( ( resolve ) => setTimeout( resolve, timeout ) );
```

Comment:
This function seems more appropriate within the `utils/actions.ts` file. We can then either expose it through the `BasePage` or directly import it from the `actions.ts`.